packages/webchat/src/components/Composer.tsx

```diff
@@ -1,7 +1,7 @@
 import { observe } from 'mobx'
 import { inject, observer } from 'mobx-react'
 import React from 'react'
-import { FormattedMessage, InjectedIntlProps, injectIntl } from 'react-intl'
+import { WrappedComponentProps, injectIntl } from 'react-intl'
```

Comment:
`InjectedIntlProps` was renamed to `WrappedComponentProps`. See info buble: https://formatjs.io/docs/react-intl/upgrade-guide-3x/#typescript-support